simple-chat-back/src/db/mongoose.js

```diff
@@ -1,14 +1,10 @@
 const mongoose = require('mongoose');
 const dotenv = require('dotenv');
 
-dotenv.config();
-
+dotenv.config()
 mongoose.connect(process.env.MONGODB_CONNECTION_STRING,{
     useNewUrlParser:true,
     useCreateIndex:true,
     useFindAndModify: false,
     useUnifiedTopology: true
-})
-mongoose.connection.on('connected', ()=>{
-    console.log('Connected')
-})
+})
```

Comment:
dont mix somelines having semicolons and some not. Choose between using or not using semicolons and stick to it. hint: [Prettier](https://prettier.io/) would greatly help you